User/Activity/doc/spec.md

```diff
@@ -0,0 +1,82 @@
+# Activity
+
+## Description
+
+This entity represents the current activity performed by a User. It can be used in different
+scenarios, from modeling social activities on a web site (e.g. Federico shares a picture of his dog) to real life activities (e.g. Federico drives his car to work). The model is largely inspired by
+[https://www.w3.org/TR/activitystreams-core](https://www.w3.org/TR/activitystreams-core).
+
+The model represents user activities using the following predicate structure: `(Agent, Verb, Object*, Target*)`, where `Object` and `Target` are optional.
+
+The `Agent` is identified by the attribute `refAgent`, the `Verb` is identified by `activityType`, the `Object` is identified by `refObject`, and the `Target` is identified by `refTarget`.
+
+## Data Model
+
+A JSON Schema corresponding to this data model can be found [here](https://fiware.github.io/dataModels/User/UserActivity/schema.json).
+
++ `id` : Unique identifier.
+
++ `type` : Entity type. It must be equal to `UserActivity`.
+
++ `dateModified` : Last update timestamp of this entity.
+    + Attribute type: [DateTime](https://schema.org/DateTime)
+    + Optional
+
++ `dateCreated` : Entity's creation timestamp.
+    + Attribute type: [DateTime](https://schema.org/DateTime)
+    + Optional  
+
++ `dateActivityStarted` : Activity's start timestamp.
+    + Attribute type: [DateTime](https://schema.org/DateTime)
+    + Mandatory    
+
++ `dateActivityEnded` : Activity's end timestamp.
+    + Attribute type: [DateTime](https://schema.org/DateTime)
+    + Optional
+
++ `refAgent` : Reference to the agent performing the activity. It may be another NGSI entity or any "Agent" identified by an URI.
+    + Attribute type: [Text](https://schema.org/Text) or [https://schema.org/URL](https://schema.org/URL)
+    + Normative References: [http://schema.org/Person](http://schema.org/Person)
+    + Mandatory
+
++ `activityType` : The action performed (e.g. Drive).
+    + Attribute type: [Text](https://schema.org/Text)
+    + Normative References: [https://schema.org/Action](https://schema.org/Action), [https://www.w3.org/TR/activitystreams-vocabulary/#activity-types](https://www.w3.org/TR/activitystreams-vocabulary/#activity-types), [https://health-lifesci.schema.org/PhysicalActivityCategory](https://health-lifesci.schema.org/PhysicalActivityCategory)
+    + Mandatory
+
++ `refObject` : Reference to the object of the action (e.g. Car1). It may be another NGSI entity or any "Thing" identified by an URI.
+    + Attribute type: [Text](https://schema.org/Text) or [https://schema.org/URL](https://schema.org/URL)
```

Comment:
instead of "Text" we should say clearly that it could be a reference to another entity, please check how that is conveyed for other data models.